PC/hu_moments_gather/hu_moments_gather.cpp

```diff
@@ -0,0 +1,106 @@
+#include <iostream>
+#include <vector>
+
+#include <opencv2/highgui/highgui.hpp>
+#include "opencv2/imgproc/imgproc.hpp"
```

Comment:
We should be consistent, and use <> for files included files, as long as it's not a file located in the repo.